.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
can we also apply to develop?

Reply:
Oh I didn't understand at first. My main goal would be to run the formatters (and eventually the tests) on pull requests.

Reply:
It will works for every PR to any branch
But additional will check AFTER pushing to master

I recommend also protect `master` and `develop` branches by next rule:
* Reqire status checks to pass before merging